cmd/eksctl-anywhere/cmd/validatecreatecluster.go

```diff
@@ -0,0 +1,140 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/dependencies"
+	"github.com/aws/eks-anywhere/pkg/executables"
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/pkg/types"
+	"github.com/aws/eks-anywhere/pkg/validations"
+	"github.com/aws/eks-anywhere/pkg/validations/commandvalidations"
+	"github.com/aws/eks-anywhere/pkg/validations/createvalidations"
+	"github.com/aws/eks-anywhere/pkg/version"
+)
+
+type validateOptions struct {
+	clusterOptions
+	hardwareCSVPath       string
+	tinkerbellBootstrapIP string
+}
+
+var valOpt = &validateOptions{}
+
+var validateCreateCmd = &cobra.Command{
+	Use:          "create -f <cluster-config-file> [flags]",
+	Short:        "Validate create cluster",
+	Long:         "This command is used to validate the create cluster action",
+	PreRunE:      bindFlagsToViper,
+	SilenceUsage: true,
+	RunE:         valOpt.validateCreateCluster,
+}
+
+func init() {
+	validateCmd.AddCommand(validateCreateCmd)
+	validateCreateCmd.Flags().StringVarP(&valOpt.fileName, "filename", "f", "", "Filename that contains EKS-A cluster configuration")
```

Comment:
were you going to put this in a method to get consistent flags?

Reply:
Yeah, I'll break that into a separate PR and rebase it in if that makes it in first.

Reply:
https://github.com/aws/eks-anywhere/pull/3227